sdk/core/azure-core/src/main/java/com/azure/core/http/policy/PortPolicy.java

```diff
@@ -43,8 +43,7 @@ public PortPolicy(int port, boolean overwrite) {
             try {
                 context.getHttpRequest().setUrl(urlBuilder.setPort(port).toUrl());
             } catch (MalformedURLException e) {
-                return Mono.error(new RuntimeException(
-                    String.format("Failed to set the HTTP request port to %d.", port), e));
+                return Mono.error(new RuntimeException("Failed to set the HTTP request port to " + port + ".", e));
```

Comment:
Should this not be monoError instead so we also log the exception?

Reply:
We can change it, though I don't know of anything that uses `PortPolicy`